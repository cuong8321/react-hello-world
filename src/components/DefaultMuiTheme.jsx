import React from 'react'
import MuiThemeProvider from 'material-ui/styles/MuiThemeProvider'
import {darkBaseTheme as defaultTheme} from 'material-ui/styles/baseThemes/darkBaseTheme'
import getMuiTheme from 'material-ui/styles/getMuiTheme'

export default function DefaultMuiTheme (props) {
  return <MuiThemeProvider muiTheme={getMuiTheme(defaultTheme)}>
    {props.children}
  </MuiThemeProvider>
}
