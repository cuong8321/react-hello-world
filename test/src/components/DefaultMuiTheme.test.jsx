import './__lib__/material-ui-mocks.jsx'
import React from 'react'
import renderer from 'react-test-renderer'
import DefaultMuiTheme from '../../../src/components/DefaultMuiTheme.jsx'

test('Snapshot: components/DefaultMuiTheme.jsx', () => {
  const tree = renderer.create(<DefaultMuiTheme><div /></DefaultMuiTheme>).toJSON()
  expect(tree).toMatchSnapshot()
})
