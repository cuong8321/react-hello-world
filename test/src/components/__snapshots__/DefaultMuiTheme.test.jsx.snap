// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Snapshot: components/DefaultMuiTheme.jsx 1`] = `
<mocked-component
  muiTheme={
    Object {
      "theme": undefined,
    }
  }
>
  <div />
</mocked-component>
`;
